### PR TITLE
Task_day2

### PR DESCRIPTION
**Pull request by shilpi1958**

### CODE REVIEW

**Comment by luis-ahw2019** [comment]
Hi Shilpi,

I am going to review this pull request.

Cheers,

Luis

**Review by luis-ahw2019 — CHANGES_REQUESTED** [review]
In line 16 the "n" should be "num".

In line 23 "true" should be "True".

File: Jain/template/simple_functions.py
Comment (luis-ahw2019) [inline Jain/template/simple_functions.py]:
```suggestion
def is_prime(num):
```

**Review by luis-ahw2019 — CHANGES_REQUESTED** [review]
The two bugs reported in the preview review have been successfully fixed :-)

Nevertheless, I have just noticed other one. Good opportunity for iterating again :-)

File: Jain/template/simple_functions.py
Comment (shilpi1958) [reply Jain/template/simple_functions.py]: Done! Thanks :)

File: Jain/template/simple_functions.py
Comment (luis-ahw2019) [inline Jain/template/simple_functions.py:17]:
```suggestion
            return False
```

**Review by luis-ahw2019 — CHANGES_REQUESTED** [review]
Oh, I am sorry by not reporting all the bugs at once, but I have discovered an indentation error now.

**Comment by bsipocz** [comment]
@shilpi1958 @luis-ahw2019 - Thanks for opening the pull request and for the review(s)! Below I have provided a few comments about the *process* of the pull request and review. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. Also, because there are pending suggestions, we will not merge this PR until the end of Astro Hack Week.

@shilpi1958 - it would have been good to provide a description for the PR, and also a title that describes the content of the PR.

@luis-ahw2019 - you've referred to certain lines in your first review. Nice to see the process that later on you've discovered that it's also possible to leave those comments on the affected lines either as a comment or a change suggestion. Another minor thing is, that you did not select "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

**Comment by bsipocz** [comment]
As a closure to AstroHackWeek, we go ahead and merge this now. Thank you both.

**Pull request merged**
